src/Installer/dnup/InstallComponent.cs

```diff
@@ -3,11 +3,12 @@
 
 namespace Microsoft.DotNet.Tools.Bootstrapper
 {
-    public enum InstallMode
+    public enum InstallComponent
```

Comment:
I'm happy to change this to a class called InstallComponents which has an enum if you'd like. In case we want to have something that we can add multiple components to or sub components of the sdk.